packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -129,7 +129,7 @@ export class BundlingDockerImage {
     // different every time the Docker layer cache is cleared, due primarily to
     // timestamps.
     const hash = FileSystem.fingerprint(path, { extraHash: JSON.stringify(options) });
-    return new BundlingDockerImage(match[1], hash);
+    return new BundlingDockerImage(`sha256:${match[1]}`, hash);
```

Comment:
Can we change the regex on L120 instead? and use `match[0]` then.

https://github.com/aws/aws-cdk/blob/867221b22df0196d6be7ff4d0eca96408e4775b0/packages/%40aws-cdk/core/lib/bundling.ts#L120

Reply:
Sure, updated.